### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by SitiSchu**
Code cleanup and readability improvements

Signed-off-by: SitiSchu <admin@sitischu.com>

### CODE REVIEW

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 177
- Added 5
           

Clones added
============
- tg_bot/modules/gtranslator.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

File: tg_bot/modules/feds.py [hunk 1]
```diff
@@ -1,1820 +1,1924 @@
...
-import json
-import time
 import csv
+import json
 import os
-from time import sleep
+import re
+import time
+import uuid
+from io import BytesIO
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/feds.py]: Codacy Issue found: [Reimport 'BytesIO' (imported line 1)](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/eval.py [hunk 2]
```diff
@@ -1,44 +1,32 @@
...
 from telegram.ext import Updater, CommandHandler, run_async
 
-from telegram.error import TimedOut, NetworkError
-
 from tg_bot import dispatcher, LOGGER
 from tg_bot.modules.disable import DisableAbleCommandHandler
-from tg_bot.modules.helper_funcs.chat_status import bot_admin, can_promote, user_admin, can_pin, dev_plus
-
-from requests import get
+from tg_bot.modules.helper_funcs.chat_status import user_admin, dev_plus
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/eval.py]: Codacy Issue found: ['tg_bot.modules.helper_funcs.chat_status.user_admin' imported but unused](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/welcome.py [hunk 3]
```diff
@@ -160,29 +169,32 @@ def new_member(bot: Bot, update: Update):
 
                 keyboard = InlineKeyboardMarkup(keyb)
 
-                sent = send(update, res, keyboard, random.choice(sql.DEFAULT_WELCOME_MESSAGES).format(first=escape_markdown(first_name)))  # type: Optional[Message]
+                sent = send(update, res, keyboard, random.choice(sql.DEFAULT_WELCOME_MESSAGES).format(
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/welcome.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/eval.py (hunk 2 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/eval.py]: Codacy Issue found: [Unused user_admin imported from tg_bot.modules.helper_funcs.chat_status](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/feds.py (hunk 1 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/feds.py]: Codacy Issue found: [redefinition of unused 'BytesIO' from line 1](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

**Comment by TsunayoshiSawada** [comment]
@SitiSchu This is on master or shiken?

**Comment by SitiSchu** [comment]
shiken, but can do master if preferred

**Comment by TsunayoshiSawada** [comment]
@TsunayoshiSawada It's shiken

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 18
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 31
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 40
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

File: tg_bot/modules/admin.py [hunk 4]
```diff
@@ -128,30 +122,28 @@ def demote(bot: Bot, update: Update, args: List[str]) -> str:
                               can_pin_messages=False,
                               can_promote_members=False)
 
-        bot.sendMessage(chat.id, "Sucessfully demoted <b>{}</b>!".format(user_member.user.first_name or user_id), parse_mode=ParseMode.HTML)
+        bot.sendMessage(chat.id, f"Sucessfully demoted <b>{user_member.user.first_name or user_id}</b>!".,
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/admin.py]: Codacy Issue found: [invalid syntax](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 40
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

File: tg_bot/modules/stickers.py [hunk 5]
```diff
@@ -47,8 +44,8 @@ def getsticker(bot: Bot, update: Update):
 def kang(bot: Bot, update: Update, args: List[str]):
     msg = update.effective_message
     user = update.effective_user
-    hash = hashlib.sha1(bytearray(user.id)).hexdigest()
-    packname = "a" + hash[:20] + "_by_"+bot.username
+    _hash = hashlib.sha1(bytearray(user.id)).hexdigest()
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/stickers.py]: Codacy Issue found: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 173
- Added 4
           

Complexity increasing per file
==============================
- tg_bot/modules/stickers.py  6
         

Clones removed
==============
+ tg_bot/modules/locks.py  -2
+ tg_bot/modules/stickers.py  -2
+ tg_bot/modules/log_channel.py  -2
+ tg_bot/modules/cust_filters.py  -2
+ tg_bot/modules/warns.py  -1
+ tg_bot/modules/bans.py  -6
+ tg_bot/modules/muting.py  -2
+ tg_bot/modules/disasters.py  -24
+ tg_bot/modules/global_bans.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

File: tg_bot/modules/welcome.py [hunk 6]
```diff
@@ -160,29 +174,31 @@ def new_member(bot: Bot, update: Update):
...
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/welcome.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/eval.py [hunk 7]
```diff
@@ -105,25 +97,26 @@ def do(func, bot, update):
...
         value = stdout.getvalue()
         result = None
         if func_return is None:
             if value:
-                result = '{}'.format(value)
+                result = f'{value}'
             else:
                 try:
-                    result = '{}'.format(repr(eval(body, env)))
+                    result = f'{repr(eval(body, env))}'
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/eval.py]: Codacy Issue found: [Use of possibly insecure function - consider using safer ast.literal_eval.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/eval.py [hunk 8]
```diff
@@ -29,16 +17,17 @@
...
 import time
 import traceback
-import re
-import io
-import asyncio
-import random
-import subprocess
-import urllib
-import psutil
+from contextlib import redirect_stdout
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/eval.py]: Codacy Issue found: [Reimport 'redirect_stdout' (imported line 1)](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/welcome.py (hunk 6 above)
Comment (SitiSchu) [reply tg_bot/modules/welcome.py]: It's not for security purposes.

File: tg_bot/modules/eval.py (hunk 7 above)
Comment (SitiSchu) [reply tg_bot/modules/eval.py]: ¯\_(ツ)_/¯

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 173
- Added 3
           

Complexity increasing per file
==============================
- tg_bot/modules/stickers.py  6
         

Clones removed
==============
+ tg_bot/modules/cust_filters.py  -2
+ tg_bot/modules/locks.py  -2
+ tg_bot/modules/bans.py  -6
+ tg_bot/modules/muting.py  -2
+ tg_bot/modules/global_bans.py  -2
+ tg_bot/modules/stickers.py  -2
+ tg_bot/modules/disasters.py  -24
+ tg_bot/modules/log_channel.py  -2
+ tg_bot/modules/warns.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

File: tg_bot/modules/stickers.py (hunk 5 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/stickers.py]: Codacy Issue found: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/welcome.py (hunk 6 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/welcome.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/eval.py [hunk 9]
```diff
@@ -105,25 +95,26 @@ def do(func, bot, update):
...
         value = stdout.getvalue()
         result = None
         if func_return is None:
             if value:
-                result = '{}'.format(value)
+                result = f'{value}'
             else:
                 try:
-                    result = '{}'.format(repr(eval(body, env)))
+                    result = f'{repr(eval(body, env))}'
```
Comment (TsunayoshiSawada) [inline tg_bot/modules/eval.py]: Codacy Issue found: [Use of possibly insecure function - consider using safer ast.literal_eval.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 173
- Added 3
           

Complexity increasing per file
==============================
- tg_bot/modules/stickers.py  6
         

Clones removed
==============
+ tg_bot/modules/disasters.py  -24
+ tg_bot/modules/muting.py  -2
+ tg_bot/modules/stickers.py  -2
+ tg_bot/modules/global_bans.py  -2
+ tg_bot/modules/cust_filters.py  -2
+ tg_bot/modules/warns.py  -1
+ tg_bot/modules/log_channel.py  -2
+ tg_bot/modules/locks.py  -2
+ tg_bot/modules/bans.py  -6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

File: tg_bot/modules/stickers.py (hunk 5 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/stickers.py]: Codacy Issue found: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/welcome.py (hunk 6 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/welcome.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/eval.py (hunk 9 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/eval.py]: Codacy Issue found: [Use of possibly insecure function - consider using safer ast.literal_eval.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

**Comment by TsunayoshiSawada** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 172
- Added 3
           

Complexity increasing per file
==============================
- tg_bot/modules/stickers.py  6
         

Clones removed
==============
+ tg_bot/modules/locks.py  -2
+ tg_bot/modules/disasters.py  -24
+ tg_bot/modules/log_channel.py  -2
+ tg_bot/modules/muting.py  -2
+ tg_bot/modules/stickers.py  -2
+ tg_bot/modules/global_bans.py  -2
+ tg_bot/modules/cust_filters.py  -2
+ tg_bot/modules/warns.py  -1
+ tg_bot/modules/bans.py  -6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705&bid=17003969)

File: tg_bot/modules/stickers.py (hunk 5 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/stickers.py]: Codacy Issue found: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/welcome.py (hunk 6 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/welcome.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

File: tg_bot/modules/eval.py (hunk 9 above)
Comment (TsunayoshiSawada) [inline tg_bot/modules/eval.py]: Codacy Issue found: [Use of possibly insecure function - consider using safer ast.literal_eval.](https://app.codacy.com/gh/AnimeKaizoku/SaitamaRobot/pullRequest?prid=5181705)

**Pull request merged**